breathecode/services/slack/tests/commands/tests_student.py

```diff
@@ -0,0 +1,156 @@
+import re, urllib
+from unittest.mock import patch
+from django.urls.base import reverse_lazy
+from rest_framework import status
+from ..mixins import SlackTestCase
+import breathecode.services.slack.commands as commands
+from ...exceptions import SlackException
+from ...commands.student import execute
+
+
+class SlackTestSuite(SlackTestCase):
+    def test_slack_command___users_is_an_empty_list(self):
+        """Testing  when passing and empty list to users."""
+
+        slack_user = {'slack_id': 'name'}
+        slack_team = {'slack_id': 'team'}
+
+        self.bc.database.create(profile_academy=1,
+                                slack_user=slack_user,
+                                capability='read_student',
+                                user=1,
+                                role='potato',
+                                academy=1,
+                                slack_team=slack_team)
+
+        data = {'text': 'student', 'user_id': 'name', 'team_id': 'team', 'channel_id': 'test'}
+
+        with self.assertRaisesMessage(SlackException, 'users-not-provided'):
+            result = execute(users=[], context=data)
+
+    def test_slack_command___user_not_registered_in_a_cohort(self):
+        """Testing when user is not registered in a cohort."""
+
+        slack_user = {'slack_id': 'name'}
+        slack_team = {'slack_id': 'team'}
+
+        self.bc.database.create(profile_academy=1,
+                                slack_user=slack_user,
+                                capability='read_student',
+                                user=1,
+                                role='potato',
+                                academy=1,
+                                slack_team=slack_team)
+
+        data = {
+            'text': 'student <@fdd2325|244372eew>',
+            'user_id': 'name',
+            'team_id': 'team',
+            'channel_id': 'test'
+        }
+        with self.assertRaisesMessage(SlackException, 'cohort-user-not-found'):
+            result = execute(users=['fdd2325'], context=data)
+
+    def test_slack_command___user_registered_in_a_cohort__without_financial_status_or_educational_status(
```

Comment:
Add a case where the cohort user is register in two different cohorts

Reply:
it has been already implemented